lib/dealership.rb

```diff
@@ -0,0 +1,32 @@
+class Dealership
+  attr_reader :inventory,
+              :inventory_count
+
+  def initialize(name, location)
+    @name = name
+    @location = location
+    @inventory = []
+    @inventory_count = 0
+  end
+
+  def add_car(new_car)
+    @inventory << new_car
+    @inventory_count += 1
+  end
+
+  def has_inventory?
+    @inventory_count > 1
+  end
+
+  def cars_by_make(make)
+    selected_cars = @inventory.select do |car|
+      car.type.split(' ')[0] == make
```

Comment:
Nice job using the `.select` enumerable here. 

Did you make a method called `make` in `car.rb` that returns the make? 
Can you just call that method on `car` and then see if it `== make`? So,
```
car.make == make
```